appveyor.yml

```diff
@@ -106,6 +106,16 @@ install:
   - cmd: ECHO "Installed SDKs:"
   - ps: "ls \"C:/Program Files/Microsoft SDKs/Windows\""
 
+  # Download visual-studio environment script from ogrisel/python-appveyor-demo
+  - ps: "\
+      (New-Object Net.WebClient).DownloadFile(\
+        'https://raw.githubusercontent.com/ogrisel/python-appveyor-demo/\
+          f54ec3593bcea682098a59b560c1850c19746e10/appveyor/run_with_env.cmd', \
+        (
+          Get-Item \".\\ci\\appveyor\\run-with-visual-studio.cmd\" -Verbose\
```

Comment:
The issue is that `Get-Item` will compain with  `ItemNotFound`. 

>  full path needs to be passed.

I don't think it is a requirement from Powershell.

The onliner I copied above is known to work ..